src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -537,9 +542,51 @@ private void SetDirty(TimeSpan? maxDelay)
             return applicationData;
         }
 
+        private async Task<Dictionary<string, string>> ReorderAndPrepareData(Dictionary<KeyValueIdentifier, string> cachedInfoLogs, CancellationToken cancellationToken = default)
+        {
+            Dictionary<string, ConfigurationSetting> precedenceData = new Dictionary<string, ConfigurationSetting>();
+            Dictionary<string, string> logIdentifiers = new Dictionary<string, string>();
+
+            foreach (KeyValuePair<KeyValueIdentifier, ConfigurationSetting> kvp in _mappedData)
+            {
+                precedenceData[kvp.Key.Key] = kvp.Value;
```

Comment:
Do we need to keep track of logIdentifiers here too?

Reply:
Shouldn't be necessary I think, the `logIdentifiers` object was just to keep track of which logs to actually print out, but we only log for values that are tracked by the change watchers.